lib/github_advisory_sync.rb

```diff
@@ -337,6 +337,20 @@ def update(package)
       package.filename
     end
 
+    def unaffected_versions_for(package)
+      if (vulnerability = vulnerabilities.first)
+        vulnerable_version_range = vulnerabilities['vulnerableVersionRange']
```

Comment:
Guess this should be "vulnerable_version_range = vulnerability['vulnerableVersionRange']".

Reply:
Thank you, fixed in 2153ee03804c017910c99c602f967f03893d8d14.